orc8r/lib/go/protos/directoryd_helper.go

```diff
@@ -70,6 +70,16 @@ func (m *GetHWIDForSgwCTeidRequest) Validate() error {
 	return nil
 }
 
+func (m *GetHWIDForSgwUTeidRequest) Validate() error {
+	if m == nil {
+		return errors.New("request cannot be nil")
+	}
+	if m.Teid == "" {
+		return errors.New("request teid cannot be empty")
```

Comment:
nit: It may be easier to debug if you change 'request" to "GetHWIDForSgwUTeidRequest"

Reply:
in this case all the functions on htis file do not include the name, just the error. The caller does include the name of the functions, so for debugging purposes it is more useful to know the caller of this function than the specific Validate function